### PR TITLE
Finished Heroku Deployment Configurations

### PR DESCRIPTION
**Pull request by lazypanda10117**

### CODE REVIEW

File: glottolog3/__init__.py [hunk 1]
```diff
@@ -56,6 +57,7 @@ def main(global_config, **settings):
     """ This function returns a Pyramid WSGI application.
     """
     settings.update(CFG)
+    settings['sqlalchemy.url'] = os.environ.get('GLOTTOLOG_DATABASE_URL');
```
Comment (baieric) [inline glottolog3/__init__.py]: Make sure to only update this setting if the environment variable exists, don't want to set this to null.

File: release.ini [hunk 2]
```diff
@@ -24,7 +22,7 @@ blog.password =
 
 [server:main]
 use = egg:waitress#main
-host = 127.0.0.1
+host = 0.0.0.0
```
Comment (baieric) [inline release.ini]: don't replace development.ini with release.ini. Both should exist, and development.ini should be unchanged.

Anything actually hosting this app will be set to use release.ini. For our development purposes, we will use development.ini

File: run.sh [hunk 3]
```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+set -e
+python setup.py develop
```
Comment (baieric) [inline run.sh]: what does the arg `develop` do?

File: serve.py [hunk 4]
```diff
@@ -0,0 +1,9 @@
+import os
+
+from paste.deploy import loadapp
+from waitress import serve
+
+if __name__ == "__main__":
+    port = int(os.environ.get("PORT", 5000))
+    app = loadapp('config:release.ini', relative_to='.')
+    serve(app, host='0.0.0.0', port=port)
```
Comment (baieric) [inline serve.py]: should the host be hard-coded? don't worry about this if it's not a quick fix

**Review by baieric — CHANGES_REQUESTED** [review]
great work, main issue is making the app still usable for development.

File: serve.py (hunk 4 above)
Comment (lazypanda10117) [reply serve.py]: this one i follow instruction from the pyramid cookbook, so I don't really know how to do otherwise for now.

File: run.sh (hunk 3 above)
Comment (lazypanda10117) [reply run.sh]: to set up the dependencies so we can run the app, I think. Also, for actual deployment, we should use install, but I am not sure on this yet. However, for it to run for demo purpose, develop will be sufficient.

**Review by baieric — APPROVED** [review]
LGTM

**Pull request merged**